lib/Twig/Extension.php

```diff
@@ -76,4 +76,12 @@ public function getGlobals()
     {
         return array();
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return '';
```

Comment:
This should probably return the FQCN to not break existing third party code that relies on the extension name.